packages/ui/src/app/pages/Validators/ValidatorList.stories.tsx

```diff
@@ -113,4 +143,78 @@ export default {
 
 type Story = StoryObj<typeof ValidatorList>
 
-export const Statistics: Story = {}
+export const StatisticsAndLists: Story = {}
+
+export const TestsFilters: Story = {
+  play: async ({ canvasElement, step }) => {
+    const screen = within(canvasElement)
+
+    const searchElement = screen.getByPlaceholderText('Search')
+    const verificationFilter = screen.getAllByText('Verification')[0]
+    const stateFilter = screen.getAllByText('State')[0]
+    const clearFilter = async () => {
+      expect(screen.getByText('Clear all filters'))
+      await userEvent.click(screen.getByText('Clear all filters'))
+    }
```

Comment:
Same here `getByText` is synchronous so no need for the `await` but IMO calling `userEvent.click(screen.getByText('Clear all filters'))` everywhere instead of `clearFilter()` is better